### PR TITLE
A few cleanup items

### PR DESCRIPTION
**Pull request by coreydaley**
Cleaning up a few items that I noticed during previous work:

- guard against Platform.AWS being nil
- error string should not start with a capital letter
- variable should not mask imported package name

All are in their own commits for easy review

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,4 +8,4 @@ COPY --from=builder /go/src/github.com/openshift/cluster-image-registry-operator
 RUN useradd cluster-image-registry-operator
 USER cluster-image-registry-operator
 COPY deploy/image-references deploy/0* /manifests/
-LABEL io.openshift.release.operator true
+LABEL io.openshift.release.operator=true
```
Comment (bparees) [inline Dockerfile]: this is not needed and makes us inconsistent with every other dockerfile in every other operator repo.  please undo this change.

Comment (coreydaley) [reply Dockerfile]: updated in current push

File: pkg/clusterconfig/clusterconfig.go [hunk 2]
```diff
@@ -103,8 +103,9 @@ func GetAWSConfig() (*Config, error) {
 		return nil, err
 	}
 	cfg.Storage.Type = StorageTypeS3
-	cfg.Storage.S3.Region = installConfig.Platform.AWS.Region
-
+	if installConfig.Platform.AWS != nil {
```
[the remark this answers is not in the document]
Comment (bparees) [reply pkg/clusterconfig/clusterconfig.go]: GetAWSConfig() should only be called when you want to use s3 storage  (meaning you already determined via some other mechanism that you want to use the s2 driver+configuration).

getPlatformStorage is the mechanism for determining that currently.

[the remark this answers is not in the document]
Comment (bparees) [reply pkg/clusterconfig/clusterconfig.go]: (so the e2e test should be updated)

[the remark this answers is not in the document]
Comment (bparees) [reply pkg/clusterconfig/clusterconfig.go]: @coreydaley please fix the e2e test in this PR.

[the remark this answers is not in the document]
Comment (coreydaley) [reply pkg/clusterconfig/clusterconfig.go]: updated in current push

File: test/e2e/aws_test.go [hunk 3]
```diff
@@ -122,10 +122,11 @@ func TestAWS(t *testing.T) {
 	tagShouldExist := map[string]string{
 		"tectonicClusterID": installConfig.ClusterID,
 	}
-	for k, v := range installConfig.Platform.AWS.UserTags {
-		tagShouldExist[k] = v
+	if installConfig.Platform.AWS != nil {
```
Comment (bparees) [inline test/e2e/aws_test.go]: this check is redundant due to the skip you added earlier.

**Comment by bparees** [comment]
/hold cancel
/lgtm

a redundant check never killed anyone.

**Comment by coreydaley** [comment]
@bparees had to rebase (had a conflict) can I get an lgtm please?
